master/buildbot/process/build.py

```diff
@@ -17,6 +17,7 @@
 
 from functools import reduce
 
+from future.utils import string_types
```

Comment:
we need to stick to what is automated with isort.
I would be incline to put future as part of the stdlib group. So we would need to add it to the config in the force stdlib section